### PR TITLE
New CartoDBfy exception and handling on import

### LINKED ISSUE #6762: CartoDBfication exceptions in importer workflow

### PR DESCRIPTION
**Pull request by iriberri**
Closes #6762 

Related to changes in: https://github.com/CartoDB/cartodb-postgresql/pull/202

Merge docs: https://github.com/CartoDB/docs/pull/727

### CODE REVIEW

**Comment by iriberri** [comment]
Test pending.

**Comment by iriberri** [comment]
Hah, the Rails test is not passing because it's dependent on the code of the cartodbfication. @rafatower could you please review? :-)

File: spec/models/table_spec.rb [hunk 1]
```diff
@@ -1751,6 +1751,17 @@ def @data_import.data_source=(filepath)
...
     end
 
+    it "should raise an error if cartodb_id is not valid" do
+      table = new_table :name => nil, :user_id => @user.id
+      table.migrate_existing_table = "cdbfytable"
+
+      @user.db_service.run_pg_query("CREATE TABLE cdbfytable (cartodb_id VARCHAR)")
+      @user.db_service.run_pg_query("INSERT INTO cdbfytable (cartodb_id) VALUES ('nan')")
+      expect {
+        table.cartodbfy
```
Comment (rafatower) [inline spec/models/table_spec.rb]: At this level what you want to test is that if cartodbfy raises an 'Invalid cartodb_id' error at DB level then the error is managed in such a way them user gets proper feedback.

You don't want to check from here that the exception is raised when fed with a wrong table. That's already done in the extension code. Actually you'd need to mock `table.cartodbfy` behavior

**Comment by iriberri** [comment]
@rafatower good to go? I guess this is a harmless deploy. :-)

**Comment by rafatower** [comment]
:+1:

**Pull request merged**